modules/tensor_mechanics/src/materials/Compute2DIncrementalStrain.C

```diff
@@ -0,0 +1,52 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
```

Comment:
Are we putting an empty line after the header or not? I'd prefer this to be consistent.